dolby_vision/src/rpu/extension_metadata/primaries.rs

```diff
@@ -50,6 +50,23 @@ pub enum MasteringDisplayPrimaries {
     SGamut3Cine,
 }
 
+impl MasteringDisplayPrimaries {
+    pub fn u8_to_alias(value: u8) -> Option<&'static str> {
```

Comment:
It would be better to implement `impl From<u8> for MasteringDisplayPrimaries` and then `impl std::fmt::Display`
so that we can do
```rs
MasteringDisplayPrimaries::from(value).to_string()
```